### ISSUE TITLE
VoiceOver does not announce contents of CodeSnippet on FireFox

### ISSUE THREAD

**Issue by daniel-schulz-privat** [carbon-design-system/carbon#6787]
## Environment

> Operating system
MacOS 10.15.5

> Browser
Firefox 80.0.1 (64-bit)

> Assistive technology used to verify
VoiceOver

## Detailed description

> What version of the Carbon Design System are you using?
what is currently exposed with https://www.carbondesignsystem.com

> What did you expect to happen?
I expect the contents of the single line CodeSnippet component to be announced

> What happened instead?
The contents was not announced

## Steps to reproduce the issue

1. browse to https://www.carbondesignsystem.com
2. select Components > CodeSnippet
3. Select 'Single Line' CodeSnippet
4. enable VoiceOver (System Preferences > Accessibility > VoiceOver > Enable VoiceOver)
5. Navigate to the Single Line CodeSnippet component using VoiceOver controls
6. try to announce contents of CodeSnippet

<img width="1190" alt="image" src="https://user-images.githubusercontent.com/20154051/92383393-86b3fc00-f10e-11ea-8f1d-b3018d927831.png">

## Additional information
- It seems that the contents is announced on both Safari and Chrome.
- It seems the contents is announced for both Multiline and inline CodeSnippeet component on Fifrefox
